### DISCUSSION TITLE
What values are expected to be filled in in SMTP-App?

### DISCUSSION THREAD

**Discussion by WeT-Klb** [nocodb/nocodb#1030] [Q&A] [answered]
Can you tell me what values are expected in the marked fields?
Usually i need Port 587 TLS or 465 SSL.

![SMTP-Setup](https://user-images.githubusercontent.com/20600342/152232514-2e50524d-236b-4907-9cd1-597161da0bd9.png)

**Comment by wingkwong** [comment, marked as answer]
Either ``true`` or ``false``. 

secure – if true the connection will use TLS when connecting to server. If false (the default) then TLS is used if server supports the STARTTLS extension. In most cases set this value to true if you are connecting to port 465. For port 587 or 25 keep it false.

ignoreTLS – if this is true and secure is false then TLS is not used even if the server supports STARTTLS extension.

To avoid confusion, we will make it as checkbox field instead.

**Reply by WeT-Klb** [reply]
Great - now it works.
